CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ Changelog
 - Implemented sync progress and payment requests with new JS api as first step to remove the purescript api ([PR 437](https://github.com/input-output-hk/daedalus/pull/437))
 - Speed optimizations for page reloads while running tests by loading bundled up for one-time tests runs ([PR 448](https://github.com/input-output-hk/daedalus/pull/445))
 - Optimized structure and naming of theming files ([PR 453](https://github.com/input-output-hk/daedalus/pull/453))
+- Added testnet label, loaded from translations for release candidate ([PR 454](https://github.com/input-output-hk/daedalus/pull/454))
```

Comment:
@darko-mijic this is a wrong URL to the PR.
Correct one is: https://github.com/input-output-hk/daedalus/pull/460